app/views/utilities/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Local Utilites</h1>
+
```

Comment:
We will be expanding this to have each utility's name to be a link to their show page, right?

Reply:
Yes, I think that we chatted about that being our way forward.